api/handler.go

```diff
@@ -336,6 +341,52 @@ func (h *apiHandler) queryUsage() http.HandlerFunc {
 	}
 }
 
+func (h *apiHandler) queryUsersUsage() http.HandlerFunc {
+	return func(rw http.ResponseWriter, r *http.Request) {
+		var (
+			from, err1 = parseInputTimestamp(r.URL.Query().Get("from"))
+			to, err2   = parseInputTimestamp(r.URL.Query().Get("to"))
+		)
+		if errs := nonNilErrs(err1, err2); len(errs) > 0 {
+			respondError(rw, http.StatusBadRequest, errs...)
+			return
+		}
+
+		var userIds []string
+		if err := json.NewDecoder(r.Body).Decode(&userIds); err != nil {
+			respondError(rw, http.StatusBadRequest, err)
+			return
+		}
```

Comment:
Won't this be called for every user in practice? I imagined we could just do a `GROUP BY` on the usage query and return the usage from ALL users instead